sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -106,6 +107,8 @@ export class Receiver {
     renewMessageLock(lockTokenOrMessage: string | ServiceBusMessage): Promise<Date>;
     }
 
+export { RetryOptions }
```

Comment:
I see `RetryOptions` being exported from the Service Bus library here, but it is not clear which of the APIs is now allowing the use of `RetryOptions` by the user

Reply:
This hsould be clear once https://github.com/azure/azure-sdk-for-js/issues/4376 is picked up